core/libs/commonwealth/commonwealth/utils/DHCPServerManager.py

```diff
@@ -3,13 +3,18 @@
 import subprocess
 from typing import Any, List, Optional, Union
 
+import psutil
 from loguru import logger
 
 
 class Dnsmasq:
-    def __init__(self, config_path: pathlib.Path) -> None:
+    def __init__(self, config_path: pathlib.Path, interface: str) -> None:
         self._subprocess: Optional[Any] = None
 
+        if interface not in psutil.net_if_stats():
+            raise ValueError(f"Interface '{interface}' not found.")
```

Comment:
It would be nice to have a list of valid interfaces on the error message.

Reply:
Done